testdata/dnn/tensorflow/generate_tf_models.py

```diff
@@ -486,6 +486,10 @@ def keras_relu6(x):
 save(sess.graph.get_tensor_by_name('keras_relu6_input:0'),
      sess.graph.get_tensor_by_name('keras_relu6/clip_by_value:0'), 'keras_relu6', optimize=False)
 ################################################################################
+inp = tf.placeholder(tf.float32, [2, 3], 'input')
```

Comment:
Will it correctly work with non-OpenCV backends?

Reply:
Locally it works on `DLIE`, `NGRAPH`, and `OCV` backends.